lua/dbee/ui/editor/init.lua

```diff
@@ -47,6 +47,7 @@ function EditorUI:new(handler, result, opts)
     buffer_options = vim.tbl_extend("force", {
       buflisted = false,
       swapfile = false,
+      filetype = "sql",
     }, opts.buffer_options or {}),
```

Comment:
This shouldn't affect users who set their own filetype in their configs, because the user configs should override the default config.